terraform/lib/dependabot/terraform/file_updater.rb

```diff
@@ -81,6 +88,58 @@ def update_registry_declaration(new_req, old_req, updated_content)
         end
       end
 
+      def update_lockfile_declaration(dependency)
```

Comment:
It looks like this method can return an unmodified lockfile in which case we'll still add it to the `updated_files` array. I wonder if this method should instead return the updated lockfile content or nil instead and then conditionally build the `updated_file(file: lockfile, content: content)` and add it to the `updated_files` array on line `31` to prevent adding it in cases where it doesn't update.

I'm thinking of the case where we no manifests change and we attempt a lockfile update but for some reason it doesn't update anything so we end up creating a PR for an unchanged lockfile.

Reply:
Good catch!